test/kernel_test.rb

```diff
@@ -816,10 +816,10 @@ def to_d
       end
 
       describe 'when a rational number is given' do
-        let(:input) { Rational(0.3) }
+        let(:input) { Rational(0.3).to_s }
```

Comment:
@AlfonsoUceda Transforming the input in a string, changes the purpose of this test. See the line above `"when a rational number is given"`.